test/e2e/gameserver_test.go

```diff
@@ -491,13 +491,16 @@ func TestGameServerReserve(t *testing.T) {
 	assert.Equal(t, "ACK: RESERVE\n", reply)
 
 	gs, err = framework.WaitForGameServerState(readyGs, agonesv1.GameServerStateReserved, time.Minute)
-	assert.NoError(t, err)
-	assert.Equal(t, agonesv1.GameServerStateReserved, gs.Status.State)
+	assert.NoError(t, err, readyGs.ObjectMeta.Name)
```

Comment:
I'm not exactly sure how this prints, but something like `fmt.Sprintf("ready game server name = %s", readyGs.ObjectMeta.Name)` might make this easier to understand in the failure output.

Reply:
Done on both!